linera-execution/src/runtime.rs

```diff
@@ -485,6 +537,8 @@ where
     }
 
     async fn write_batch_and_unlock(&self, batch: Batch) -> Result<(), ExecutionError> {
+        let size = batch.size() as u64;
+        self.bytes_write.fetch_add(size, Ordering::Relaxed);
```

Comment:
Should we compare to the write limit here, and return an error if it's too much?

Reply:
Yes, we can and I did it.